modules/http/src/main/java/org/mule/module/http/internal/request/HttpResponseToMuleEvent.java

```diff
@@ -108,9 +108,18 @@ else if (responseContentType.startsWith(APPLICATION_X_WWW_FORM_URLENCODED.toLowe
         }
 
 
-        MuleMessage message = new DefaultMuleMessage(muleEvent.getMessage().getPayload(), inboundProperties,
+        DefaultMuleMessage message = new DefaultMuleMessage(muleEvent.getMessage().getPayload(), inboundProperties,
                                                      null, inboundAttachments, muleContext, muleEvent.getMessage().getDataType());
 
+
+        for (String property : inboundProperties.keySet())
```

Comment:
This is the second time i've seen this right after DefaultMuleMessage constructor, is there no way it can be part of the constructor logic?